packages/core/src/redux/slices/remoteParticipants.ts

```diff
@@ -305,9 +305,10 @@ export const doRequestAudioEnable = createAppAuthorizedThunk(
  */
 
 export const selectRemoteParticipantsRaw = (state: RootState) => state.remoteParticipants;
-export const selectRemoteParticipants = (state: RootState) => state.remoteParticipants.remoteParticipants;
+export const selectRemoteClients = (state: RootState) => state.remoteParticipants.remoteParticipants;
 
-export const selectNumParticipants = createSelector(
-    selectRemoteParticipants,
-    (clients) => clients.filter((c) => !NON_PERSON_ROLES.includes(c.roleName)).length + 1,
+export const selectRemoteParticipants = createSelector(selectRemoteClients, (clients) =>
+    clients.filter((c) => !NON_PERSON_ROLES.includes(c.roleName)),
 );
+
+export const selectNumParticipants = createSelector(selectRemoteParticipants, (clients) => clients.length + 1);
```

Comment:
Is this +1 correct? If we are counting human participants, then the local participant might be non-human. Dont know exactly where this selector is used, but imo we should have `selectNumClients` and `selectNumParticipants` to align with the two other selectors

Reply:
Yeah, good call. Fixed now. Added a selector for `numClients`, and checking the role of the local participant in the `numParticipant` selector.